feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/model/Input.kt

```diff
@@ -0,0 +1,48 @@
+package com.hedvig.android.odyssey.model
+
+import com.hedvig.android.odyssey.repository.AutomationClaimDTO2
+import com.hedvig.android.odyssey.repository.AutomationClaimInputDTO2
+import com.hedvig.odyssey.remote.file.File
+import com.hedvig.odyssey.remote.money.MonetaryAmount
+
+/**
+ * An Input describes what the user needs to update the claim with in order to continue the automation flow.
+ * Map the input fields to the corresponding fields on ClaimState when updating the claim.
+ */
+sealed interface Input {
```

Comment:
These are the ones that are supposed to be generated from the GQL Schema that odyssey will expose right?

Reply:
I was thinking of these as an internal representation, so we can hopefully transform what we get from odyssey to this type. If not we have to rethink this a bit!

Reply:
Right, yeah good idea!